pennylane/templates/subroutines/multiplier.py

```diff
@@ -41,24 +41,27 @@ class Multiplier(Operation):
 
     .. math::
 
-        \text{Multiplier}(k,mod) |x \rangle = | x \cdot k \; \text{modulo} \; \text{mod} \rangle.
+        \text{Multiplier}(k,mod) |x \rangle = | x \cdot k \; \text{mod} \; mod \rangle.
+
+    This operation can be represented in a quantum circuit as:
 
     The implementation is based on the quantum Fourier transform method presented in
     `arXiv:2311.08555 <https://arxiv.org/abs/2311.08555>`_.
 
     .. note::
 
-        Note that :math:`x` must be smaller than :math:`mod` to get the correct result. Also, it
-        is required that :math:`k` has inverse, :math:`k^-1`, modulo :math:`mod`. That means
-        :math:`k*k^-1 modulo mod is equal to 1`, which will only be possible if :math:`k` and
-        :math:`mod` are coprime. Furthermore, if :math:`mod \neq 2^{len(x\_wires)}`, two more
-        auxiliaries must be added.
+        To obtain the correct result, :math:`x` must be smaller than :math:`mod`. Also, it
+        is required that :math:`k` has inverse, :math:`k^{-1}`, modulo :math:`mod`. That means
+        :math:`k \cdot k^{-1}` modulo :math:`mod` is equal to 1, which will only be possible if :math:`k` and
+        :math:`mod` are coprime.
+
+    .. seealso:: :class:`~.PhaseAdder` and :class:`~.OutMultiplier`.
 
     Args:
         k (int): the number that needs to be multiplied
         x_wires (Sequence[int]): the wires the operation acts on
-        mod (int): the modulus for performing the multiplication, default value is :math:`2^{len(x\_wires)}`
-        work_wires (Sequence[int]): the auxiliary wires to be used for performing the multiplication
+        mod (int): the modulo for performing the multiplication. If not provided, it will be set to :math:`2^{\text{len(output_wires)}}`.
```

Comment:
```suggestion
        mod (int): the modulo for performing the multiplication. If not provided, it will be set to :math:`2^{\text{len(output_wires)}}`
```